deblur/deblurring.py

```diff
@@ -52,7 +52,7 @@ def get_sequences(input_seqs):
     return seqs
 
 
-def deblur(input_seqs, read_error=0.05, mean_error=None, error_dist=None,
+def deblur(input_seqs, read_error=0.05, mean_error=0.005, error_dist=None,
```

Comment:
the read error is the upper bound on the error rate, whereas the mean error is the mean. They should not be the same. It was a mistake to make them identical (when i run deblurring i don't use the default values, so didn't notice it until now...)